client/src/Components/App/App.js

```diff
@@ -33,7 +33,10 @@ const App = () => {
       activity: activity,
       journal_entry: journal
     };
-    fetchCall(newEntry).then(updateLogs);
+    fetchCall(newEntry).then(() => {
+      updateLogs();
+      setUser({id: 1, name: 'KB'});
+    });
```

Comment:
Thank you for updating this!! Also allows for a fairly quick plug and play on the front end if/when we add user authentication